src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -134,8 +134,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         dispatcher.RunAsync(Windows::UI::Core::CoreDispatcherPriority::Normal, [weakThis{ get_weak() }]() {
             if (auto strongThis{ weakThis.get() })
             {

```

Comment:
You're gonna want: [whitespace off](https://github.com/microsoft/terminal/pull/13876/files?w=1)